packages/pages/src/dev/server/server.ts

```diff
@@ -37,6 +38,9 @@ export const createServer = async (dynamicGenerateData: boolean) => {
     displayGenerateTestDataWarning = !(await generateTestData());
   }
 
+  // Redirect urls with a final slash to their canonical url without the slash
+  // app.use(finalSlashRedirect);
```

Comment:
uncomment ?

Reply:
I think you were looking at an old commit. This is not commented in the latest commit.